packages/contract-wrappers/src/contract_wrappers/coordinator_wrapper.ts

```diff
@@ -0,0 +1,855 @@
+import { CoordinatorContract, CoordinatorRegistryContract, ExchangeContract } from '@0x/abi-gen-wrappers';
+import { getContractAddressesForNetworkOrThrow } from '@0x/contract-addresses';
+import { Coordinator, CoordinatorRegistry, Exchange } from '@0x/contract-artifacts';
+import { schemas } from '@0x/json-schemas';
+import { generatePseudoRandomSalt, signatureUtils } from '@0x/order-utils';
+import { Order, SignedOrder, SignedZeroExTransaction, ZeroExTransaction } from '@0x/types';
+import { BigNumber, fetchAsync } from '@0x/utils';
+import { Web3Wrapper } from '@0x/web3-wrapper';
+import { ContractAbi } from 'ethereum-types';
+import * as HttpStatus from 'http-status-codes';
+import * as _ from 'lodash';
+
+import { orderTxOptsSchema } from '../schemas/order_tx_opts_schema';
+import { txOptsSchema } from '../schemas/tx_opts_schema';
+import { CoordinatorTransaction, OrderTransactionOpts } from '../types';
+import { assert } from '../utils/assert';
+import {
+    CoordinatorServerApprovalRawResponse,
+    CoordinatorServerApprovalResponse,
+    CoordinatorServerCancellationResponse,
+    CoordinatorServerError,
+    CoordinatorServerErrorMsg,
+    CoordinatorServerResponse,
+} from '../utils/coordinator_server_types';
+import { decorators } from '../utils/decorators';
+import { TransactionEncoder } from '../utils/transaction_encoder';
+
+import { ContractWrapper } from './contract_wrapper';
+/**
+ * This class includes all the functionality related to filling or cancelling orders through
+ * the 0x V2 Coordinator extension contract.
+ */
+export class CoordinatorWrapper extends ContractWrapper {
```

Comment:
Add a CHANGELOG entry to `contract-wrappers` detailing the addition of this wrapper.